src/app/amsterdam/projecten/stadsdeel/page.tsx

```diff
@@ -23,7 +23,9 @@ function Project() {
       </Grid>
       <Grid gapVertical="large" paddingBottom="x-large">
         <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }}>
-          <Heading level={1}>Bouw- en verkeersprojecten in Noord</Heading>
+          <Heading level={1} className="ams-mb-m">
```

Comment:
Please add this margin to the ‘Bouw- en verkeersprojecten’ heading in `projecten/page.tsx` as well.